src/renderer/views/Watch/Watch.js

```diff
@@ -1534,6 +1542,21 @@ export default Vue.extend({
       return timestamp
     },
 
+    keyboardShortcutHandler: function (event) {
+      if (event.ctrlKey || document.activeElement.classList.contains('ft-input')) {
+        return
+      }
+      switch (event.key) {
```

Comment:
this won't work if caps lock is on and you hit shift
```suggestion
      switch (event.key.toUpperCase()) {
```

Reply:
Do we want it to though?
The Youtube shortcuts behave the same as the current implementation.
With <kbd>CapsLock</kbd> enabled, pressing <kbd>N</kbd> and <kbd>P</kbd> goes to next/prevous video, and if you do it while holding down <kbd>Shift</kbd> it doesn't do.

Reply:
I'm getting the opposite result when testing on YouTube

Reply:
Hmm alright. I'll go ahead and add your suggestions.